src/data.ts

```diff
@@ -24,17 +24,26 @@ export interface JsonEvent extends DateOrRange {
   confirmed?: boolean;
 }
 
+export interface JsonImage {
+  url?: string;
+  title?: string;
+}
+
 /** Json representation of an individual. */
 export interface JsonIndi {
   id: string;
   firstName?: string;
   lastName?: string;
+  maidenName?: string;
   famc?: string;
   fams?: string[];
+  numberOfChildren?: number;
+  numberOfMarriages?: number;
   birth?: JsonEvent;
   death?: JsonEvent;
   sex?: string;
-  imageUrl?: string;
+  mainImage?: JsonImage;
```

Comment:
I'm not convinced a separate `mainImage` field is necessary. It should be enough to have one `images` field. We can always simply take the first element from the image array as the main image.